opensrp-chw-malaria/src/test/java/org/smartregister/activity/BaseMalariaProfileActivityTest.java

```diff
@@ -57,4 +57,14 @@ public void formatTime() {
         }
     }
 
+    @Test
+    public void setupView() throws Exception {
+        BaseMalariaProfileActivity baseMalariaProfileActivity = Mockito.spy(BaseMalariaProfileActivity.class);
+        Mockito.doNothing().when(baseMalariaProfileActivity).initializeFloatingMenu();
```

Comment:
@ladleo this test does not seem to cover the anc/pnc visits functions

Reply:
But the test isn't for the visitations, I am testing setupViews() which happens to call my visit functions

Reply:
So I am verifying that when setupViews() run, so will my anc and pnc functions